# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Watchtower.cpp
# Commit: 8129ee5398d5
# Full Hash: 8129ee5398d5ce34183dd2ca51c3aa1d27e9c318
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2023-01-20 03:46:23
# Regressor Bug: 1809359
# File Overlap Count: 1
# Description:
#   Bug 1809359 - Inline SetElementMegamorphic r=jandem
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D166401
# ==============================================================================

diff -r a4156a9d0b2b -r 8129ee5398d5 js/src/vm/Watchtower.cpp
--- a/js/src/vm/Watchtower.cpp	Thu Jan 19 22:22:32 2023 +0000
+++ b/js/src/vm/Watchtower.cpp	Thu Jan 19 23:53:40 2023 +0000
@@ -96,6 +96,7 @@
   MOZ_ASSERT(obj->isUsedAsPrototype());
 
   cx->caches().megamorphicCache.bumpGeneration();
+  cx->caches().megamorphicSetPropCache.bumpGeneration();
 }
 
 // static
